src/main/java/com/coveros/selenified/Capabilities.java

```diff
@@ -267,8 +267,11 @@ private Boolean runHeadless() {
      * @param extraCapabilities any additional parameters to set for selenium
      */
     public void addExtraCapabilities(DesiredCapabilities extraCapabilities) {
+        setDesiredCapabilities();
         if (extraCapabilities != null && browser.getName() != BrowserName.NONE) {
             desiredCapabilities = desiredCapabilities.merge(extraCapabilities);
+            // look for specifics
+            desiredCapabilities.setJavascriptEnabled(extraCapabilities.is("javascriptEnabled"));
```

Comment:
shouldn't the .merge(extraCapabilities take care of this?

Reply:
I think this was leftover from some old code. I've removed it, and put in a UT to verify it still works